clappr/clappr-ios
# Create Fullscreen plugin

⚠️ DO NOT MERGE THIS PR UNTIL [THIS PR](https://github.com/clappr/clappr-ios/pull/227) IS MERGED ⚠️

### Goal

Creating a Fullscreen plugin to append in MediaControl plugins

### How to test

Run example and tests: 

1. Make sure that Fullscreen functionality conform to business rules
2. Make sure to test Fullscreen controlled by app and by player
3. Make sure to test entering in Fullscreen mode
